src/components/policy/src/sql_pt_representation.cc

```diff
@@ -644,7 +644,6 @@ bool SQLPTRepresentation::GatherConsumerFriendlyMessages(
 
   if (query.Prepare(sql_pt::kCollectFriendlyMsg)) {
     while (query.Next()) {
-
       UserFriendlyMessage msg;
       msg.message_code = query.GetString(7);
       std::string language = query.GetString(6);
```

Comment:
@AByzhynar , I didn't check it. I had just removed an empty line with using **check_style.sh** for success build on Travis CI.
I think adding **const** in that file should be checked in another task